src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java

```diff
@@ -230,6 +247,43 @@ public boolean getDefaultFormProcessing() {
         };
     }
 
+    public boolean addOtherSRS(ResourceInfo resourceInfo) {
+        // do nothing when
+        if (resourceInfo.getStore().getType() == null) return false;
+        else if (!resourceInfo.getStore().getType().equalsIgnoreCase("Web Feature Server (NG)"))
+            return false;
+        else if (!(resourceInfo instanceof FeatureTypeInfo)) return false;
+
+        try {
+            // featureType.
+            FeatureTypeInfo featureType = (FeatureTypeInfo) resourceInfo;
+            // for WFS-NG resource store OTHER_SRS in metadata
+
+            Name nativeName = new NameImpl(featureType.getNativeName());
+
+            org.geotools.data.wfs.internal.FeatureTypeInfo info =
+                    (org.geotools.data.wfs.internal.FeatureTypeInfo)
+                            featureType
+                                    .getStore()
+                                    .getDataStore(null)
+                                    .getFeatureSource(nativeName)
+                                    .getInfo();
+            // read all identifiers of this CRS into a an comma seperated string
+            if (info.getOtherSRS() != null) {
+                if (!info.getOtherSRS().isEmpty()) {
+                    featureType
+                            .getMetadata()
+                            .put(FeatureTypeInfo.OTHER_SRS, String.join(",", info.getOtherSRS()));
+                    return true;
+                }
+            }
+        } catch (IOException e) {
+            Logger.getGlobal().log(Level.SEVERE, e.getMessage(), e);
+        }
+
+        return false;
+    }
+
```

Comment:
this function is used to read other srs and update metadata map of FeautreTypeInfo, during Resource creation/modification on UI